back-end/API/app/dtos/enterprise_dto.py

```
@@ -0,0 +1,25 @@
+from pydantic import BaseModel, ConfigDict, Field, EmailStr
+
+class _EterpriseBase(BaseModel):
+    name: str = Field(min_length=3, max_length=255)
+    email: EmailStr = Field(max_length=255)
+    phone: str = Field(min_length=3, max_length=255)
+    address: str = Field(min_length=3, max_length=255)
+    cityId: int
+
+class EnterpriseCreateDTO(_EterpriseBase):
+    isTemporary: bool = False
+    pass
```

Ruff (0.15.13)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)